CourseGrab/Home/HomeViewController.swift

```diff
@@ -9,43 +9,68 @@
 import UIKit
 import Tactile
 
+enum TableSection {
+    case available([Section]), awaiting([Section])
+}
```

Comment:
can we move this inside `HomeViewController` and make it `private` as well. We don't want other view controllers to be using this and its likely that other view controllers will also have its own `TableSection`

Reply:
But if we make it private then won't `HomeTableViewHeader` not be able to access it?

Reply:
oh yeah i forgot about that. ignore making it private